Contents/Code/__init__.py

```diff
@@ -485,11 +485,27 @@ def compile_metadata(self, helper):
             tagger.add_authors_to_moods()
         # Series.
         tagger.add_series_to_moods()
-        # Setup title + subtitle where available.
-        if helper.subtitle:
-            album_title = helper.title + ': ' + helper.subtitle
+
+        # If the `simplify_title` option is selected, don't append subtitle
+        # and remove extra endings on the title
+        if Prefs['simplify_title']:
```

Comment:
This would be cleaner as a function in the class `AlbumUpdateTool` from `update_tools.py`. I believe this because it is doing more than a simple check and assign. It should be enough to name the new helper function `simplify_title`. And have the function return the simplified title.

Then you can call the new helper function from __init__.py like
```
# Some comment
simplified_title = helper.simplify_title()
if simplified_title:
    album_title = simplified_title
else if helper.subtitle:
    album_title = helper.title + ': ' + helper.subtitle
else:
     album_title = helper.title

Reply:
So I moved this to the `AlbumUpdateTool` but I'm not sure if I did it exactly like you wanted?  

```py
if Prefs['simplify_title']:
    album_title = helper.simplify_title()
elif helper.subtitle:
    album_title = helper.title + ': ' + helper.subtitle
else:
    album_title = helper.title
```

Does this work?

Reply:
That's perfect!